docs/en/dgs/tutorial.md

```diff
@@ -1,38 +1,56 @@
 # Tutorial
 
-This document is a turotial of offline training and online inference for a GNN model with:
-  - GraphLearn,
-  - Dynamic graph service,
+This document is an e2e tutorial of offline training and online inference for a GNN model with:
+  - GraphLearn-Training(gl) for offline training.
+  - Dynamic-Graph-Service(dgs) for online inference.
   - Tensorflow model serving.
 
-Here is an example of a supervised job with EgoSage, containing the following sections.
+Here is an example of a supervised job with EgoBipartiteSage, containing the following sections.
 1. Prepare data, including bulk-loading data for offline training and streaming data for online inference.
-2. Train the EgoGage model using the offline bulk-loading data.
+2. Train the EgoBipartiteSage model using the offline bulk-loading data.
 3. Exporting TF model.
 4. Deploying TF model on tensorflow model serving.
-4. Deploy the online graph sampling service.
-5. Start Java Client, sample and predict.
+5. Deploy the online dynamic graph service and ingest the streaming data.
+6. Start Java Client, sample and predict.
 
-The 1-3 parts are envolved in GraphLearn-Traing, ref to [GraphLearn-Training](../gl/intro.md) for details.
 
-## Prepare data
-We start with cora as an example, online deployment with simulated data.
+## 1. Prepare data
+
+We provide a script to generate data with [u2i](https://github.com/alibaba/graph-learn/blob/master/dynamic_graph_service/conf/u2i/schema.u2i.json) schema.
+The u2i graph contains two vertex type: `user` and `item`, and two edge type: `u2i`(from `user` to `item`) and `i2i`(from `item` to `item`).
+Run this script:
+
 ```shell
-cd examples/data
-python cora.py
+cd dynamic_graph_service
+python3 python/data/u2i/u2i_generator.py --output-dir /tmp/u2i_gen
 ```
 
-TODO(@Seventeen17): use open streaming data source
+This generator with first create a static graph for bulk-loading data, which will be used for offline training.
+After that, the streaming data will consume the bulk-loading data with timestamp 0 as an initial state, and further generate graph updates on an increased timestamp.
```

Comment:
While generating the streaming data, the previously generated bulk-loading data will be used as the initial state of the graph whose timestamp set to 0.  After that, we generate more graph updates with increasing timestamps.

Reply:
done